clang/test/CodeGen/distributed-thin-lto/thinlto-distributed-backend-skip.ll

```diff
@@ -6,7 +6,7 @@
 ; RUN: opt -thinlto-bc -o %t.o %s
 
 ; RUN: %clang_cc1 -triple x86_64-grtev4-linux-gnu \
-; RUN:   -fthinlto-index=%S/Inputs/thinlto-distributed-backend-skip.bc \
+; RUN:   -fthinlto-index=%S/../Inputs/thinlto-distributed-backend-skip.bc \
```

Comment:
Why is this change needed?

Reply:
Because the test was moved down a directory, but the checked in sharded summary used as input to the test hasn't.